concent_api/core/tests/utils.py

```diff
@@ -67,6 +68,14 @@ def add_time_offset_to_date(base_time: str, offset: int) -> str:
     )
 
 
+def generate_uuids(number_of_uuids: int) -> List[str]:
```

Comment:
Can you give an example of where this might be used to generate more than one id at a time ? I can think of `task_id` and `subtask_id` which are sometimes generated line by line, but maybe it would be simpler to just call it twice ?

Also, why not returning tuple ?

Reply:
quite frankly I don't see any, but it was @cameel 's idea. 
@cameel ?

As to returning tuple - what would be the advantage over list? Lists can be easily produced by means of list comprehension, that's way I have chosen it .

Reply:
ok, I chenged this function so it now returns just a single UUID, with the possibility of changing the last character

Reply:
My idea was a bit different. I agree with @rwrzesien that creating a list and having to choose one is not very convenient.

My idea was to have a function like this:
```
def generate_uuid(mnemonic: str) -> str
    ...
```

UUIDs are very hard to remember and recognize. Unless you know one by heart, you have to just copy and paste them every time. @dybi suggested the natural thing to do: create a constant. Since we may sometimes need more than one I suggested the natural extension: let's create a function that returns the same UUID given the same mnemonic. Then, if you need two UUIDs you could simply do `generate_uuid('provider'), generate_uuid('requestor')` instead of having to remember and copy long strings of characters.

Simple and elegant. Does not require you to create constants if you just need a predictable UUID for testing. And implementation is dead simple. I have originally suggested using a number as a mnemonic and simply replacing a part of the UUID and recalculating the checksum. But there's an even easier way. [UUID versions 3 and 5](https://en.wikipedia.org/wiki/Universally_unique_identifier#Versions_3_and_5_(namespace_name-based)) are meant exactly for that. Just use [`uuid5()`](https://docs.python.org/3.7/library/uuid.html#uuid.uuid5). We could even just use it directly without wrapping it in our own helper.

Reply:
So it will generate same UUID each time for a given value ?

Reply:
Yeah. I that's how it's supposed to work:


``` python
from uuid import UUID, uuid5

>>> uuid5(UUID('00000000-0000-0000-0000-000000000000'), 'provider')
UUID('d81821ad-2d8e-58a5-a74a-05c272696d84')
>>> uuid5(UUID('00000000-0000-0000-0000-000000000000'), 'provider')
UUID('d81821ad-2d8e-58a5-a74a-05c272696d84')
>>> uuid5(UUID('00000000-0000-0000-0000-000000000000'), 'provider')
UUID('d81821ad-2d8e-58a5-a74a-05c272696d84')

>>> uuid5(UUID('00000000-0000-0000-0000-000000000000'), 'requestor')
UUID('894cf831-772d-5ec1-9c2f-ed0cc997b3d0')
>>> uuid5(UUID('00000000-0000-0000-0000-000000000000'), 'requestor')
UUID('894cf831-772d-5ec1-9c2f-ed0cc997b3d0')
>>> uuid5(UUID('00000000-0000-0000-0000-000000000000'), 'requestor')
UUID('894cf831-772d-5ec1-9c2f-ed0cc997b3d0')
```